pkg/cloudprovider/kubevirt/loadbalancer_test.go

```diff
@@ -483,6 +486,31 @@ func TestEnsureLoadBalancer(t *testing.T) {
 			"cluster1",
 			"192.168.0.38",
 			nil,
+			nil,
+		},
+		// Testcase: No LB exists for service, single node given, annotation on original service
```

Comment:
Got it, thanks! I was misunderstanding the structure of `EXPECT`. The semantics are the same as Java but the syntax is different and I wasn't reading the error messages properly, in part because of messages such as [this](https://github.com/kubevirt/cloud-provider-kubevirt/blob/df5ad0c0cda8a3c8aaaa6c48a32fb4d25a786d15/pkg/cloudprovider/kubevirt/loadbalancer.go#L70). So I thought the error emitted was relevant when it was an artifact of the test. My bad, this branch started with a stale pull and then got crazy from there. 

I went with your suggestion to merge it with the first test case data and change the service object match. Thank you for your kindness in looking this over for me!